.ko.yaml

```diff
@@ -1,182 +1,200 @@
+---
 defaultBaseImage: cgr.dev/chainguard/alpine-base:latest
+baseImageOverrides:
+  github.com/sigstore/scaffolding/cmd/cloudsqlproxy: gcr.io/cloudsql-docker/gce-proxy:1.32.0-alpine
```

Comment:
should we use 1.33.1 here, as that is the default in the helm chart

https://github.com/sigstore/helm-charts/blob/fdca671033e29dc58e15234644cf511d146f72db/charts/trillian/Chart.yaml#LL37C14-L37C118

Reply:
yes, thanks for the catch!

Reply:
done, PTAL